aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -493,6 +493,13 @@ module aptos_framework::voting {
         voting_forum.next_proposal_id
     }
 
+    #[view]
+    public fun get_proposer<ProposalType: store>(voting_forum_address: address, proposal_id: u64): address acquires VotingForum {
+        let voting_forum = borrow_global<VotingForum<ProposalType>>(voting_forum_address);
+        let proposal = table::borrow(&voting_forum.proposals, proposal_id);
```

Comment:
Minor nit: I would probably put this in an inline function to be used on all of them, though it looks like it's already done like this elsewhere

Reply:
i can follow up in a separate PR to change all of them

Reply:
done